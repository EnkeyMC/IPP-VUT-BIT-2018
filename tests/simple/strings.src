.IPPcode18

defvar GF@a
defvar GF@b
defvar GF@res

move GF@a string@Hello
move GF@b string@World

concat GF@res GF@a GF@b
write GF@res

strlen GF@res GF@a
write GF@res

getchar GF@res GF@b int@2
write GF@res

setchar GF@a int@1 GF@res
write GF@a
