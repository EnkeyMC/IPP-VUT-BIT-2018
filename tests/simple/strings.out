HelloWorld
5
r
Hrllo
